src/api/ClassAPI.ts

```diff
@@ -1,7 +1,10 @@
 import * as APIUtils from "./APIUtils";
-import { ClassDetailResponse } from "./types";
+import { ClassDetailResponse, ClassRequest } from "./types";
 
 const getClass = (id: number): Promise<ClassDetailResponse> =>
   APIUtils.get(`/classes/${id}`) as Promise<ClassDetailResponse>;
 
-export default { getClass };
+const putClass = (data: ClassRequest): Promise<ClassDetailResponse> =>
+  APIUtils.put(`/classes/${data.id}/`, data);
```

Comment:
there's a wee compilation error complaining about the type here - i think we just need to add something like this:
```suggestion
  APIUtils.put(`/classes/${data.id}/`, data) as Promise<ClassDetailResponse>;
```